Automattic/wc-calypso-bridge
# Update menu item colors to match Calypso

Updates the menu items to match Calypso's current colors and styling and fix CSS conflicts in core.

### Before
<img width="281" alt="Screen Shot 2019-07-24 at 4 22 50 PM" src="https://user-images.githubusercontent.com/10561050/61798420-daf11380-ae5b-11e9-8794-85e98232ee68.png">

### After
<img width="278" alt="Screen Shot 2019-07-24 at 9 39 48 PM" src="https://user-images.githubusercontent.com/10561050/61798424-dd536d80-ae5b-11e9-9095-afbc7925fc02.png">

### Testing
1.  Visit various pages around the dashboard.
2.  Make sure the menu is correctly styled and active colors matches those in Calypso.
